keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/ResourceControllerTest.kt

```diff
@@ -50,17 +48,27 @@ internal class ResourceControllerTest {
   @Autowired
   lateinit var resourceRepository: ResourceRepository
 
+  @Autowired
+  lateinit var resourcePersister: ResourcePersister
```

Comment:
I found the thing. If you just annotate this `@MockBean` you don't need the extra configuration class.